x-pack/plugins/cross_cluster_replication/public/app/sections/home/home.js

```diff
@@ -85,6 +87,7 @@ export const CrossClusterReplicationHome = injectI18n(
                   onClick={() => this.onSectionChange(tab.id)}
                   isSelected={tab.id === this.state.activeSection}
                   key={tab.id}
+                  data-test-subject={tab.dataTestSubj}
```

Comment:
We can update this logic to treat anything defined on `tab` as pass-through props, so we can define `data-test-subj` on the tab config objects, but maybe that's just taking things too far. 😄 

```jsx
<EuiTabs>
  {this.tabs.map(tab => {
    const { id, name, ...rest } = tab;
    return (
      <EuiTab
        onClick={() => this.onSectionChange(id)}
        isSelected={id === this.state.activeSection}
        key={id}
        {...rest}
      >
        {name}
      </EuiTab>
    );
  })}
</EuiTabs>
```

Reply:
If you want to leave it as-is I'd also suggest renaming it to `testSubj` because the data part is just a side effect of the HTML spec around `data-` attributes.